frontend/src/components/Pages/MissionPage/TaskOverview/TaskTable.tsx

```diff
@@ -7,8 +7,12 @@ import { tokens } from '@equinor/eds-tokens'
 import { getColorsFromTaskStatus } from 'utils/MarkerStyles'
 
 const StyledTable = styled(Table)`
-    grid-column: 1/ -1;
-    font: equinor;
+    display: grid;
+    overflow: auto;
+    @media (max-width: 700px) {
+        width: calc(80vw);
+    }
+    max-width: 950px;
```

Comment:
```suggestion
    max-width: calc(80vw);
```

Reply:
![image](https://github.com/equinor/flotilla/assets/33518988/3df266ac-218f-49e7-b8d3-c104a75f7546)
If not we can get this where width is forcing the table bigger than it wants to be

Reply:
I have made this change now